src/layout/msWorkspace/msWorkspace.ts

```diff
@@ -130,7 +130,6 @@ export class MsWorkspace extends WithSignals {
         this.layout.onDestroy();
         if (this.msWorkspaceActor) {
             this.msWorkspaceActor.destroy();
-            delete this.msWorkspaceActor;
```

Comment:
We are destroying the MsWindow anyway, so all references it has will be gone with it (assuming we actually remove all references to the MsWindow). We can't just delete the msWorkspaceActor field because the type does not include `undefined`, and changing it to include `undefined` adds a bunch of unnecessary checks everywhere.